packages/next/src/routes/rest/routeError.ts

```diff
@@ -73,14 +72,14 @@ export const routeError = async ({
   collection,
   config: configArg,
   err,
-  req,
+  req: incomingReq,
 }: {
   collection?: Collection
   config: Promise<SanitizedConfig> | SanitizedConfig
   err: APIError
-  req: Partial<PayloadRequest>
+  req: PayloadRequest | Request
```

Comment:
When would this be `Request` ?

Reply:
Because we don't always pass `PayloadRequest` to `routeError`
https://github.com/payloadcms/payload/blob/e72f12af9765db6f0bf877597edca5f8844c3f71/packages/next/src/routes/rest/index.ts#L201-L228

In `routeError` we cast `incomingReq` to `PayloadRequest` then:
https://github.com/payloadcms/payload/blob/6cc06052f03e5bf1d73c752e9ee52e7b72c627f5/packages/next/src/routes/rest/routeError.ts#L82-L99